packages/@webex/contact-center/src/types.ts

```diff
@@ -0,0 +1,108 @@
+export enum HTTP_METHODS {
+  GET = 'GET',
+  POST = 'POST',
+  PATCH = 'PATCH',
+  PUT = 'PUT',
+  DELETE = 'DELETE',
+}
+
+export type WebexRequestPayload = {
+  method?: HTTP_METHODS;
+  uri?: string;
+  addAuthHeader?: boolean;
+  headers?: {
+    [key: string]: string | null;
+  };
+  body?: object;
+  statusCode?: number;
+  json?: boolean;
+};
+
+type Listener = (e: string, data?: unknown) => void;
+type ListenerOff = (e: string) => void;
+
+export type ServiceHost = {
+  host: string;
+  ttl: number;
+  priority: number;
+  id: string;
+  homeCluster?: boolean;
+};
+
+export interface CCConfig {
+  cc: {
```

Comment:
Why is the config nested within cc ? Is this part of a larger config block?

Reply:
Used it similarly how it was done for meetings, no specific reason

Reply:
I think we can remove this since the interface is already CC_CONFIG. Please add an underscore between the words

Reply:
done